misc/scripts/install-local.sh

```diff
@@ -252,23 +252,29 @@ function get_incompatible_releases() {
 }
 
 function is_compatible_arch() {
-    local input=("${@}")
+    local input=("${@}") ret=1
```

Comment:
```suggestion
    local input=("${@}") ret=1 pacarch farch
```

Reply:
Why would I localize those? They’re fill-ins for a for loop.